book/templates.rst

```diff
@@ -1,200 +1,721 @@
+--------------------------
 
-                <tag name="sulu.rlp"/>
-            </property>
+In your projects, you will need several templates for different parts of your
+website. The easiest way is to copy and adjust the default template.
+
+The first thing you need to adjust is the ``<key>``. This is the unique
+identifier of the template:
+
+.. code-block:: xml
+
+    <!-- app/Resources/templates/pages/event.xml -->
+    <?xml version="1.0" ?>
+    <template xmlns="http://schemas.sulu.io/template/template"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
 
-            <property name="images" type="media_selection">
+        <key>event</key>
+
+        <!-- ... -->
+    </template>
+
+The second thing you have to customize is the ``<view>``. This element stores
+the Twig file that is used to render the template:
+
+.. code-block:: xml
+
+    <!-- app/Resources/templates/pages/event.xml -->
+    <?xml version="1.0" ?>
+    <template xmlns="http://schemas.sulu.io/template/template"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
+        <!-- ... -->
+
+        <view>ClientWebsiteBundle:templates:event</view>
+
+        <!-- ... -->
+    </template>
+
+.. Note::
+
+    The notation ``bundle:directory:filename`` is Symfony's naming convention
+    for Twig templates. Read `more about template naming`_ in the Symfony
+    documentation.
+
+We'll talk more about the Twig file itself in :doc:`twig`. Let's continue with
+adding properties to our page template.
+
+Properties
+----------
+
+Properties make up the structure of a page. They are defined in the element
+``<properties>``:
+
+.. code-block:: xml
+
+    <!-- app/Resources/templates/pages/event.xml -->
+    <?xml version="1.0" ?>
+    <template xmlns="http://schemas.sulu.io/template/template"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
+        <!-- ... -->
+
+        <properties>
+            <!-- ... -->
+
+            <property name="startDate" type="date">
                 <meta>
-                    <title lang="en">Images</title>
+                    <title lang="en">Start Date</title>
                 </meta>
             </property>
 
-            <property name="article" type="text_editor">
+            <!-- ... -->
+        </properties>
+    </template>
+
+A property has three essential attributes:
+
+* a ``name`` that is unique within a template
+* a ``type`` that defines what kind of content can be stored
+* a ``title`` that is shown in the administration interface
+
+Here is a table with the content types shipped in Sulu core:
```

Comment:
i think we should mention here the `sulu:content:types:dump` which lists all available content-types in the current system.